starwars/src/App.js

```diff
@@ -1,19 +1,38 @@
-import React from 'react';
-import './App.css';
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import StarWarsCard from "./components/StarWarsCard.js";
+import "./App.css";
 
 const App = () => {
+  const [peopleData, setPeopleData] = useState(null);
+  useEffect(() => {
+    axios
+      .get("https://swapi.co/api/people")
+      .then(res => {
+        console.log(res.data.results);
+        setPeopleData(res.data.results);
+      })
+      .catch(function(error) {
+        console.log(error);
+      });
+  }, []);
   // Try to think through what state you'll need for this app before starting. Then build out
   // the state properties here.
 
-  // Fetch characters from the star wars api in an effect hook. Remember, anytime you have a 
+  // Fetch characters from the star wars api in an effect hook. Remember, anytime you have a
   // side effect in a component, you want to think about which state and/or props it should
   // sync up with, if any.
 
   return (
     <div className="App">
       <h1 className="Header">React Wars</h1>
+      {peopleData ? (
+        peopleData.map(x => <StarWarsCard character={x} />)
```

Comment:
Make sure to add a `key ` value to your components liske so :
` peopleData.map(( x, i ) => <StarWarsCard character={x}  key={i}/>)`